packages/core-data/src/entities.js

```diff
@@ -207,6 +210,15 @@ export const rootEntitiesConfig = [
 		baseURLParams: { context: 'edit' },
 		plural: 'globalStylesVariations', // Should be different than name.
 		getTitle: ( record ) => record?.title?.rendered || record?.title,
+		getRevisionsUrl: ( parentId, revisionId ) =>
+			`/wp/v2/global-styles/${ parentId }/revisions${
+				revisionId ? '/' + revisionId : ''
+			}`,
+		revisionURLParams: { context: 'view' },
+		supports: {
+			revisions: true,
+		},
+		supportsPagination: true,
```

Comment:
What is this config about? is it related to this PR?

Reply:
Yes and no 😄  

The flag was [introduced](https://github.com/WordPress/gutenberg/commit/da1290d5820a2ef24176d1b87ade0a602f00e141#diff-38a74d088ea018dbbea00d8f5bd5c03ea54bfc099d1d1836686c24a5582bf579R123) in https://github.com/WordPress/gutenberg/pull/55164 as a check before adding the `'X-WP-Total'` header value to meta.

Should global styles behave differently do you think?

Reply:
Oh I forgot I added this, since it never made it to core, maybe we should move it to the new "support" key as well for consistency? or if we have other similar flags, move everything top level?